test/e2e/adminUI/pages/fieldTypes/url.js

```diff
@@ -0,0 +1,39 @@
+var utils = require('../../../utils');
+
+module.exports = function UrlType(config) {
+	var self = {
+		selector: '.field-type-url[for="' + config.fieldName + '"]',
+		elements: {
+			label: '.FormLabel',
+			value: 'input[name="' + config.fieldName + '"]',
+		},
+		commands: [{
```

Comment:
You fixed all cases?  There were a few.  Also regarding that merge thing you get I think you should make sure you sync/rebase your branch with master before you PR.  Also it's also a good idea to squash all the less significant commits so they don't pollute the history.